assets/wix/Product.wxs

```diff
@@ -317,6 +338,36 @@
   </UI>
 </Fragment>
 
+<!-- Microsoft Update Menu Dialog -->
+<Fragment>
+ <UI>
+    <Property Id="USE_MU" Value="1" />
+    <Property Id="ENABLE_MU" Value="1" />
+    <Dialog Id="MuDialog" Width="370" Height="270" Title="!(loc.ExitDialog_Title)">
+      <!-- Banner -->
+      <Control Id="BannerBitmap" Type="Bitmap" X="0" Y="0" Width="370" Height="44" TabSkip="yes" Text="!(loc.InstallDirDlgBannerBitmap)"/>
+      <Control Id="Title" Type="Text" X="15" Y="6" Width="200" Height="30" Transparent="yes" NoPrefix="yes" Text="{\WixUI_Font_Title}Use Microsoft Update to help keep your computer secure and up to date" TabSkip="yes" />
+      <Control Id="BannerLine" Type="Line" X="0" Y="44" Width="370" Height="0" TabSkip="yes" />
+      <Control Id="Description" Type="Text" X="25" Y="54" Width="280" Height="100" Transparent="yes" NoPrefix="yes" Text="Microsoft Update helps keep your computer secure and up to date for Windows and other Microsoft products, including PowerShell 7. Updates will be delivered based on your current update settings. You can review or change these settings from the Windows Update control panel." TabSkip="yes" />
+      <!-- If the checkboxes are defined first, then they are first in the tab order and can be ticked and unticked using the spacebar -->
+      <Control Id="UseMuCheckBox" Type="CheckBox" X="20" Y="100" Width="290" Height="27" Property="USE_MU" CheckBoxValue="0" Text="Enable updating PowerShell through Microsft Update or WSUS (recommended)"/>
+      <Control Id="EnableMuCheckBox" Type="CheckBox" X="20" Y="130" Width="290" Height="17" Property="ENABLE_MU" CheckBoxValue="0" Text="Use Microsoft Update when I check for updates (recommended)"/>
+      <Control Id="MuFAQ" Type="Hyperlink" X="20" Y="190" Width="214" Height="17">
+        <Text><![CDATA[<a href="https://aka.ms/PowerShell-Microsoft-Update-FAQ">See the Microsoft Update FAQ.</a>]]></Text>
```

Comment:
This link takes me to a BING homepage, doesn't contain any Microsoft Update information (as far as I can tell)

Reply:
We'll need to update the link, but this can be done after the get the code committed.

Reply:
Filed a doc request for this:  https://github.com/MicrosoftDocs/PowerShell-Docs/issues/7787